librocksdb-sys/build.rs

```diff
@@ -357,9 +355,40 @@ fn update_submodules() {
     }
 }
 
+fn update_submodules() {
+    let program = "git";
+    let dir = "../";
+    let args = ["submodule", "update", "--init"];
+    run_command(program, dir, &args);
+}
+
+fn cherry_pick_commits(commits: &Vec<&str>) {
```

Comment:
```suggestion
fn cherry_pick_commits(commits: &[&str]) {
```

Reply:
I don't know to to get a slice from a split function